tests/unit/test_helpers.py

```diff
@@ -13,3 +13,5 @@ def test_to_lower_camel_case():
     assert helpers.to_lower_camel_case('CamelCaseStr') == 'camelCaseStr'
     assert helpers.to_lower_camel_case('lowerCamelCaseStr') == 'lowerCamelCaseStr'
     assert helpers.to_lower_camel_case('snake_case_str') == 'snakeCaseStr'
+    assert helpers.to_lower_camel_case('_start_with_underscore') == 'StartWithUnderscore'
+    assert helpers.to_lower_camel_case('__start_with_double_underscore') == 'StartWithDoubleUnderscore'
```

Comment:
Can we add more tests for edge cases? Are we failing gracefully?

Reply:
Added. The only possible fail if an argument of given method is not the string, in this case we shouldn't fail gracefully, imho